frontend/smart-kickers-game/src/App.js

```diff
@@ -1,17 +1,29 @@
 import { useEffect, useState } from 'react';
 import './App.css';
-import { resetGame } from './apis/Game';
 import { Button } from './components/Button';
 import GameResults from './components/GameResults.js';
+import { resetGame } from './apis/resetGame';
+import GameStatistics from './components/GameStatistics.js';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faPerson } from '@fortawesome/free-solid-svg-icons';
 
 import config from './config';
 
+library.add(faPerson);
+
 function App() {
   const [blueScore, setBlueScore] = useState(0);
   const [whiteScore, setWhiteScore] = useState(0);
+  const [toggleGameScore, setToggleGameScore] = useState(false);
 
+  const [
+    finalScores = {
+      blue: 0,
+      white: 0,
+    },
+    setFinalScores,
+  ] = useState({ blue: 0, white: 0 });
```

Comment:
Do you really need the new state for that? My proposal is to remove that and move it to props (check second comments)

Reply:
We need to have it, this will prevent user from scoring goals in statistics view (the stats should not be changed after end game button is pressed).  We tried your proposed solution but the score was keep changing.